enhancements/sig-policy/128-resource-selector/README.md

```diff
@@ -0,0 +1,144 @@
+# Resource selector for `ConfigurationPolicy`
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io/)
+
+## Summary
+
+Add a resource selector to pair with the existing `namespaceSelector` in order to allow a
+`ConfigurationPolicy` to filter and select objects when an `objectDefinition` doesn't have a name
+provided. The resource selector would also enable the enforcement of `objectDefinitions` without a
+name.
+
+## Motivation
+
+The `ConfigurationPolicy` currently has a `namespaceSelector` to configure the controller to filter
+and range over namespaces relevant to a particular namespaced `objectDefinition`. Furthermore,
+currently an `objectDefinition` without a name is feasible but is not enforceable. A
+`resourceSelector` would both pair well with the `namespaceSelector` for further object filtering as
+well as enable an enforceable unnamed `objectDefinition` by providing the controller with a list of
+names to handle.
+
+Adding a `resourceSelector` would lower the bar to entry for `ConfigurationPolicy`, adding an
+additional configuration available to users without requiring knowledge of Go templating as is
+required with the alternative, `object-templates-raw`.
+
+### Goals
+
+- Add a `resourceSelector` to `ConfigurationPolicy` to allow the controller to handle objects
+  without requiring `object-templates-raw`
+- Consolidate compliance messages to list out objects per namespace (one downside to the alternative
+  `object-templates-raw` is there's a compliance message for each object)
+- Make unnamed `objectDefinition`s enforceable
+- Make the `namespaceSelector` apply to `Namespace`s when a `resourceSelector` is not provided for a
+  `Namespace` `objectDefinition`, allowing a related `Namespace` to be created alongside related
+  namespaced objects.
+
+### Non-Goals
+
+- Establishing complex logic in the `resourceSelector` that could not be handled readily by the
+  Kubernetes API and, by extension, the `kubernetes-dependency-watches` library.
```

Comment:
This is about the "include" and "exclude" glob filters we have, is that right? I just want to check that there isn't some other complex logic we've considered.

Reply:
Yes, the include/exclude glob filters.